app-modules/division/database/migrations/2023_09_15_201951_create_divisions_table.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+return new class () extends Migration {
+    public function up(): void
+    {
+        Schema::create('divisions', function (Blueprint $table) {
+            $table->uuid('id')->primary();
+
+            $table->string('name');
+            $table->string('code')->unique(); //unique?
```

Comment:
The `unique` on the `code` make sense to me, so we can remove the comment.

I also think we should add `unique` on the `name` as well.

What do you think @dgoetzit?

Reply:
@Orrison Yeah I think that makes sense.